src/client.rs

```diff
@@ -44,7 +47,7 @@ impl Client {
     /// Get and verify a specific coin.
     pub async fn get_coin(
         &mut self,
-        header: Header,
+        _header: Header,
```

Comment:
just did this a few places where the variables were not used to remove warnings

Reply:
Sure, though obviously eventually that argument will be used. Right now we don't really verify a lot of things that need to be verified